nrf_modem/README.rst

```diff
@@ -3,7 +3,7 @@
 Modem library
 #############
 
-The Modem library is the primary interface for operating the nRF9160 modem, establishing the LTE-M and NB-IoT connections, and receiving the position data (GPS).
+The Modem library is the primary interface for operating the cellular modem, establishing the LTE-M and NB-IoT connections, and receiving the position data (GPS).
```

Comment:
Could we have said the `nRF91 series modem` here?

Reply:
@divipillai

Reply:
Agree, we could use the "nRF91 Series modem" term here. Could see similar terms used in the infocenter docs for nRF9160.

Reply:
Updated